HousingFinanceInterimApi/V1/UseCase/LoadSuspenseCashTransactionsUseCase.cs

```diff
@@ -0,0 +1,128 @@
+            IBatchLogErrorGateway batchLogErrorGateway,
+            IUPCashLoadSuspenseAccountsGateway upCashLoadSuspenseAccountsGateway,
+            IGoogleFileSettingGateway googleFileSettingGateway,
+            IGoogleClientService googleClientService)
+        {
+            _batchLogGateway = batchLogGateway;
+            _batchLogErrorGateway = batchLogErrorGateway;
+            _upCashLoadSuspenseAccountsGateway = upCashLoadSuspenseAccountsGateway;
+            _googleFileSettingGateway = googleFileSettingGateway;
+            _googleClientService = googleClientService;
+        }
+
+        public async Task<StepResponse> ExecuteAsync()
+        {
+            LoggingHandler.LogInfo($"Starting suspense cash import");
+
+            const string sheetName = "Cash";
+            const string sheetRange = "A:E";
+
+            var batch = await _batchLogGateway.CreateAsync(CashSuspenseLabel).ConfigureAwait(false);
+            var googleFileSettings = await GetGoogleFileSetting(CashSuspenseLabel).ConfigureAwait(false);
+
+            if (googleFileSettings == null)
+                return new StepResponse() { Continue = false, NextStepTime = DateTime.Now.AddSeconds(0) };
+
+            var allSuspenseTransactions = await _googleClientService
+                .ReadSheetToEntitiesAsync<CashSuspenseTransactionAuxDomain>(googleFileSettings.GoogleIdentifier, sheetName, sheetRange)
+                .ConfigureAwait(false);
+
+            if (allSuspenseTransactions.Any())
+            {
+                var filledNewAccounts = allSuspenseTransactions.Where(x => !string.IsNullOrEmpty(x.NewRentAccount)).ToList();
+
+                await HandleSpreadSheet(batch.Id, filledNewAccounts).ConfigureAwait(false);
+            }
+
+            var suspenseTransactions = await _upCashLoadSuspenseAccountsGateway.GetCashSuspenseTransactions().ConfigureAwait(false);
+
+            List<IList<object>> rows = new List<IList<object>>();
+
+            //HEADER
+            rows.Add(new List<object>() { "Id", "Original Payment Ref", "Payment Date", "Amount", "New Payment Ref" });
+
+            //ROWS            
+            foreach (var suspenseTransaction in suspenseTransactions)
+            {
+                rows.Add(new List<object>() {
+                    suspenseTransaction.Id,
+                    suspenseTransaction.RentAccount,
+                    suspenseTransaction.Date.ToString("dd/MM/yyyy"),
+                    suspenseTransaction.Amount,
+                    "" });
+            }
+
+            await _googleClientService.UpdateSheetAsync(rows, googleFileSettings.GoogleIdentifier, sheetName, sheetRange, true).ConfigureAwait(false);
+
+            await _batchLogGateway.SetToSuccessAsync(batch.Id).ConfigureAwait(false);
+            LoggingHandler.LogInfo($"End suspense cash import");
+            return new StepResponse() { Continue = true, NextStepTime = DateTime.Now.AddSeconds(int.Parse(_waitDuration)) };
```

Comment:
could improve the readability of this newing by spacing out the property assignments into multiple lines.
Better still, create a small private method that tells us the function of this new returned object?